RussianFilter/sections/adservers.txt

```diff
@@ -12,6 +12,7 @@
 ||rygfjv.com^$third-party
 ||qdb6gd3nth.ru^$third-party
 ||ofbjgf.com^$third-party
+||pecukirom.com^$third-party
```

Comment:
Are ads loaded from this domain? I think we can add it to the Spyware and Annoyances filters.